ubuntu_bug_triage/__main__.py

```diff
@@ -96,20 +101,24 @@ def launch():
     args.status = list(set(args.status))
     setup_logging(args.debug)
 
+    if not args.ignore:
+        args.ignore = []
+
```

Comment:
We can remove this by setting the default above.

Reply:
Removed